app/src/main/kotlin/com/wire/android/ui/home/conversations/common/ConversationItemTemplate.kt

```diff
@@ -0,0 +1,42 @@
+package com.wire.android.ui.home.conversations.common
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.wire.android.ui.home.conversationslist.common.EventBadgeFactory
+import com.wire.android.ui.home.conversationslist.common.RowItem
+import com.wire.android.ui.home.conversationslist.model.EventType
+
+@Composable
+fun ConversationItemTemplate(
+    leadingIcon: @Composable () -> Unit,
+    title: @Composable () -> Unit,
+    subTitle: @Composable () -> Unit = {},
+    eventType: EventType? = null,
+    onConversationItemClick: () -> Unit = { }
```

Comment:
Should `RowItem` be always clickable? If so then it shouldn't have a default empty value, because it will make a ripple effect even if there is no real action after that